### DISCUSSION TITLE
Twitch

### DISCUSSION THREAD

**Discussion by Bhaalspawn** [smplayer-dev/smplayer#37] [Q&A]
How can i watch twitch channels on SMplayer?
i copy/paste urls but they don't work

**Comment by smplayer-dev** [comment]
You also need a recent version of youtube-dl.

**Reply by Bhaalspawn** [reply]
i have, but it doesn't work

**Reply by smplayer-dev** [reply]
Open a issue then, and don't forget to include the smplayer log after trying to play a twitch url.
